SpeechToText: Add Overview demo description (#31227)
DevExpress/DevExtreme: apps/demos/Demos/SpeechToText/Overview/description.md

> Our [SpeechToText](/Documentation/ApiReference/UI_Components/dxSpeechToText) component allows you to integrate voice input within your DevExtreme-powered app. The component implements the Web Speech API [SpeechRecognition](https://developer.mozilla.org/en-US/docs/Web/API/SpeechRecognition) interface and supports [custom speech recognizers](/Documentation/ApiReference/UI_Components/dxSpeechToText/Configuration/#customSpeechRecognizer).

Reviewer: A few minor corrections. I moved DevExtreme earlier for SEO.
```suggestion
Our DevExtreme [SpeechToText](/Documentation/ApiReference/UI_Components/dxSpeechToText) component allows you to integrate voice input to your app. The component implements the [Web Speech API SpeechRecognition](https://developer.mozilla.org/en-US/docs/Web/API/SpeechRecognition) interface and supports [custom speech recognizers](/Documentation/ApiReference/UI_Components/dxSpeechToText/Configuration/#customSpeechRecognizer).
```

<!--split-->

> You can integrate SpeechToText with any text input, including other [DevExtreme components](https://js.devexpress.com/jQuery/Demos/WidgetsGallery/). To do this, set the component's **value** property to the transcribed text. SpeechToText returns transcribed text in the [onResult](/Documentation/ApiReference/UI_Components/dxSpeechToText/Configuration/#onResult) handler as users speak. When speech stops, the component calls the [onEnd](/Documentation/ApiReference/UI_Components/dxSpeechToText/Configuration/#onEnd) handler and switches from the "listening" state to the initial state. SpeechToText implements different icon ([startIcon](/Documentation/ApiReference/UI_Components/dxSpeechToText/Configuration/#startIcon)/[stopIcon](/Documentation/ApiReference/UI_Components/dxSpeechToText/Configuration/#stopIcon)), text ([startText](/Documentation/ApiReference/UI_Components/dxSpeechToText/Configuration/#startText)/[stopText](/Documentation/ApiReference/UI_Components/dxSpeechToText/Configuration/#stopText)), and click handler ([onStartClick](/Documentation/ApiReference/UI_Components/dxSpeechToText/Configuration/#onStartClick)/[onStopClick](/Documentation/ApiReference/UI_Components/dxSpeechToText/Configuration/#onStopClick)) properties in each component state.

Reviewer: Do you usually highlight component names with bold or inline code?

```suggestion
You can integrate SpeechToText into any text input (including [DevExtreme components](https://js.devexpress.com/jQuery/Demos/WidgetsGallery/)). Assign transcribed text to the component **value** property. SpeechToText returns transcribed text in the [onResult](/Documentation/ApiReference/UI_Components/dxSpeechToText/Configuration/#onResult) event handler as users speak. When speech stops, the component raises the [onEnd](/Documentation/ApiReference/UI_Components/dxSpeechToText/Configuration/#onEnd) event and switches from the "listening" state to the initial state. SpeechToText implements various icons ([startIcon](/Documentation/ApiReference/UI_Components/dxSpeechToText/Configuration/#startIcon)/[stopIcon](/Documentation/ApiReference/UI_Components/dxSpeechToText/Configuration/#stopIcon)), text ([startText](/Documentation/ApiReference/UI_Components/dxSpeechToText/Configuration/#startText)/[stopText](/Documentation/ApiReference/UI_Components/dxSpeechToText/Configuration/#stopText)) properties in each component state. You can also handle clicks ([onStartClick](/Documentation/ApiReference/UI_Components/dxSpeechToText/Configuration/#onStartClick)/[onStopClick](/Documentation/ApiReference/UI_Components/dxSpeechToText/Configuration/#onStopClick))
```

Author: No, we treat them as regular words (names).

> For a complete overview of SpeechToText options (including [Web Speech API options](/Documentation/ApiReference/UI_Components/dxSpeechToText/Configuration/speechRecognitionConfig/)), refer to the [SpeechToText API Reference](/Documentation/ApiReference/UI_Components/dxSpeechToText/).

Reviewer: ```suggestion
For a complete overview of SpeechToText options (including [Web Speech API options](/Documentation/ApiReference/UI_Components/dxSpeechToText/Configuration/speechRecognitionConfig/)), refer to the following topic: [SpeechToText API Reference](/Documentation/ApiReference/UI_Components/dxSpeechToText/).
```
